kayenta-sql/src/main/java/com/netflix/kayenta/sql/config/SqlConfiguration.java

```diff
@@ -28,4 +36,35 @@
 @ComponentScan({"com.netflix.kayenta.sql"})
 @EntityScan({"com.netflix.kayenta.sql"})
 @EnableJpaRepositories({"com.netflix.kayenta.sql"})
-public class SqlConfiguration extends DataSourceAutoConfiguration {}
+@EnableConfigurationProperties(SqlProperties.class)
+@RequiredArgsConstructor
+public class SqlConfiguration extends DataSourceAutoConfiguration {
+
+  private final SqlProperties sqlProperties;
+  private final List<StorageService> storageServices;
+
+  @Bean
+  @ConditionalOnProperty("kayenta.sql.migration.enabled")
+  public StorageDataMigrator storageDataMigrator() throws IOException {
+    var sourceStorageServiceClassName =
+        sqlProperties.getMigration().getSourceStorageServiceClassName();
+    var targetStorageServiceClassName =
```

Comment:
Given SQL storage doesn't have a concept of "account".  I'm inclined to remove the idea of a target account.    OR make this migrator more generic and independent of SQL config and let it copy data from one storage account to another of any kind.  

SQL - I'd like to remove "accounts" entirely.